src/main/java/pinger/Pinger.java

```diff
@@ -45,7 +45,9 @@ public static void main(String[] args)  {
             } 
         }while(ch == 'y');
     }
-
+    /*The readalllines() is used to take input repetatively
```

Comment:
javadoc comments should have two asterisks and should not have any text on the first line. 
```java
/**
 * docs go here
 */
```
also the `readAllLines` function reads all lines from any input stream. it isn't used to read input from the user.